operator/tests/central/basic/75-enable-central-db-external.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-central-db-password
+data:
+  password: c2VjcmV0Cg== # "secret\n"
+---
+apiVersion: platform.stackrox.io/v1alpha1
+kind: Central
+metadata:
+  name: stackrox-central-services
+spec:
+  central:
+    db:
```

Comment:
Not a code review comment, just a question: from what I understand, the `ROX_POSTGRES_DATASTORE` env var must be true for a Central to support Postgres. Are these tests executed for both cases (true and false, where RocksDB is still used) or how does this work?